onboarding/store.js

```diff
@@ -164,11 +220,18 @@ module.exports = {
   ),
   init: function (store) {
     const userString = window.localStorage.getItem(ONBOARDING_USER)
+    const botString = window.localStorage.getItem(ONBOARDING_BOT)
```

Comment:
okay, here's the comment on why we shouldn't use `localStorage` to get and set the onboarding pub:

using `localStorage` to get and set the onboarding user was an _exception_. we had an exceptional case where the user could have filled out the form to create a user, but then wasn't yet logged in with that user. if the refreshed the page, we would have no idea who they were, so we'd put them back at the beginning of the onboarding, instead of letting them know to check their email to log in.

in our case here, to get to the second step of the onboarding means you've successfully logged in, so now we know who you are and have authenticated access to the api. this means we are able to get and set your pub data in the api. (i was thinking we'd start with only allowing one pub per user, but if we want to do multiple then we'll need to know the id of the pub, which i'd recommend we add to the url once they create a pub object with an id.)

does that make sense?

Reply:
this makes sense.